bee-ledger/src/workers/error.rs

```diff
@@ -7,76 +7,57 @@ use bee_message::{address::Address, milestone::MilestoneIndex, Error as MessageE
 
 use crate::{
     types::{Balance, Error as TypesError, Unspent},
-    workers::snapshot::error::Error as SnapshotError,
+    workers::{pruning::error::PruningError, snapshot::error::Error as SnapshotError},
 };
 
 /// Errors occurring during ledger workers operations.
 #[derive(Debug, thiserror::Error)]
+#[allow(missing_docs)]
 pub enum Error {
-    /// Snapshot error.
```

Comment:
Is this PR up to date / rebased ? This is already gone in upstream.

Reply:
yea, rebased now :+1:

Reply:
I don't see it

Reply:
yea, wasn't pushed yet.